hail/python/hail/docs/_templates/layout.html

```diff
@@ -146,36 +120,33 @@
         <div class="wy-side-nav-search">
           {% block sidebartitle %}
 
+          
+          {% include "searchbox.html" %}
+
+          {% endblock %}
+        </div>
+
+        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
           {% if logo and theme_logo_only %}
             <a href="{{ pathto(master_doc) }}">
           {% else %}
-            <a href="{{ pathto(master_doc) }}" class="icon icon-home"> {{ project }}
+            <a href="{{ pathto(master_doc) }}" class="icon icon-home"> {{ project }} Docs
+            {% if theme_display_version %}
+                {%- set nav_version = version %}
+                {% if READTHEDOCS and current_version %}
+                  {%- set nav_version = current_version %}
+                {% endif %}
+                {% if nav_version %}
+                    ( {{ nav_version }} )
```

Comment:
the spaces between the version number and the parentheses looks funny, let's not have any.